.rubocop.yml

```diff
@@ -0,0 +1,36 @@
+require: rubocop-rails
```

Comment:
This is my customization to the rubocop, usually i always use those in any rails project